Makefile

```diff
@@ -105,12 +105,21 @@ test: ## Run tests
 test-integration: ## Run integration tests
 	go test -v -tags=integration ./test/integration/...
 
-.PHONY: test-e2e
-test-e2e: ## Run e2e tests
-	PULL_POLICY=IfNotPresent $(MAKE) docker-build
+.PHONY: full-capd-e2e
+full-capd-e2e: ## Rebuild all manifests and all provider images then run the capd-e2es
 	$(MAKE) generate-manifests
 	$(MAKE) release-manifests
-	cd ./test/e2e; MANAGER_IMAGE=$(CONTROLLER_IMG)-$(ARCH):$(TAG) go test -v -tags=e2e -timeout=1h . -args -ginkgo.v -ginkgo.trace
+	$(MAKE) all-images-capd-e2e
```

Comment:
Does `scripts/ci-capd-e2e.sh` need to be updated to run this?

Reply:
no, unfortunately this part is a little confusing, but these are explicitly for dev-workflows. The CI script uses the lower-level methods (docker-build) with custom environment variables for CI

Reply:
I do need to edit the docs though and the book to make the doc visible